pydantic/main.py

```diff
@@ -14,13 +14,16 @@
     Callable,
     Dict,
     List,
+    Literal,
     Mapping,
     Optional,
     Tuple,
     Type,
     TypeVar,
     Union,
     cast,
+    get_origin,
+    get_type_hints,
```

Comment:
They don't exist in python 3.7-. We'll probably add `get_origin` and `get_type_hints` directly in `pydantic.typing` to support fallback for python 3.7-

Reply:
Ok. I can look in to that

Reply:
You could have a look at #1832 and in particular this commit d86a184072655d93652d6ea941421005f34de962 that I quote in the end. But I feel like this should be done in a dedicated PR (I'll open one to add python 3.9 support soon) so maybe the best is just to wait a bit as you wish !